### ISSUE TITLE
Link with anchor target to the iframe

### ISSUE THREAD

**Issue by shenzhuxi** [PolymerElements/paper-drawer-panel#69]
I have links in paper-header-panel drawer with anchor target to an iframe in paper-header-panel main. 
The whole panel will be scrolled with the iframe, when a link is clicked. 
It only happened in webkit browsers but not Firefox.

**Comment by blasten** [comment]
@shenzhuxi can you provide a link?  I tried the following and it works as expected:

``` html
<paper-drawer-panel>
    <paper-header-panel drawer>
      <paper-toolbar>
        <div>Application</div>
      </paper-toolbar>
      <a target="iframe1" href="https://en.wikipedia.org/wiki/Toronto_St._Patricks">Link 1</a> <br />
      <a target="iframe1" href="https://en.wikipedia.org/wiki/Toronto_Maple_Leafs">Link 2</a> <br />
      <a target="iframe1" href="https://en.wikipedia.org/wiki/National_Hockey_League">Link 3</a>  <br />
      <a target="iframe1" href="https://en.wikipedia.org/wiki/Catholic_Church">Link 4</a></div>
    </paper-header-panel>
    <paper-header-panel main>
      <paper-toolbar>
        <paper-icon-button icon="menu" paper-drawer-toggle></paper-icon-button>
      </paper-toolbar>
      <div> 
        <iframe frameborder="0" name="iframe1" class="fit" height="100%" width="100%"></iframe>
      </div>
    </paper-header-panel>
</paper-drawer-panel>
```

**Comment by shenzhuxi** [comment]
It's very weird.  
I copied https://en.wikipedia.org/wiki/Catholic_Church to my web server and changed the link. 

```
  <paper-drawer-panel>
    <paper-header-panel drawer>
      <paper-toolbar>
        <div>Application</div>
      </paper-toolbar>
      <a target="iframe1" href="http://localhost/Catholic_Church.html#Western_rites">Same as localhost parent page </a></div>
      <a target="iframe1" href="http://localhost:8008/Catholic_Church.html#Western_rites">Different from the localhost parent page</a></div>
    </paper-header-panel>
    <paper-header-panel main>
      <paper-toolbar>
        <paper-icon-button icon="menu" paper-drawer-toggle></paper-icon-button>
      </paper-toolbar>
      <div> 
        <iframe frameborder="0" id="iframe1" name="iframe1" style="position: absolute;" height="100%" width="100%"></iframe>
      </div>
    </paper-header-panel>
  </paper-drawer-panel>

```

On Chrome with "position: absolute;", this problem only happens with links with the same domain name as the parent page. Did you tried on Chrome too? 
On Safari with "position: absolute;", both links have the whole panel shift problem.

**Comment by blasten** [comment]
@shenzhuxi this might happen if the height of the iframe is greater than the height of the main container. 

When you specify an anchor  such as `#Western_rites` in the URL, the browser will try to scroll to the element that has that anchor. 

If you have nested scrollable containers, you might hit this issue. e.g. if the iframe, main, and body are scrollable. 
In the example I provided, I used the `class="fit"` and I used the demo included in this repo.

Link: http://output.jsbin.com/seguni
